en/components/grids_templates/export-excel.md

```diff
@@ -104,22 +108,28 @@ The Excel Exporter service can export data to excel from the @@igxName. The data
 
 ## Exporting @@igComponent's Data
 
-To start using the IgniteUI Excel Exporter first import the [`IgxExcelExporterService`]({environment:angularApiUrl}/classes/igxexcelexporterservice.html) in the app.module.ts file and add the service to the `providers` array:
+Getting the exporters into your project takes only a few lines of code. Follow these steps and you will have reusable services that can create either Excel or PDF outputs on demand:
+
+1. Import the [`IgxExcelExporterService`]({environment:angularApiUrl}/classes/igxexcelexporterservice.html) and/or [`IgxPdfExporterService`]({environment:angularApiUrl}/classes/igxpdfexporterservice.html) in your root module.
+2. Provide the services (for earlier Angular versions) or rely on the tree-shakable providers available since v12.2.1.
+3. Inject whichever exporter you need and call its `export` method when the user requests a file.
+
+Here is the module setup in case you are targeting versions prior to v12.2.1:
```

Comment:
This isn't applicable for the `IgxPdfExporterService` as it is just about to be released. Not available in versions prior to 12.2.1.